extensions/amp-story/0.1/amp-story.css

```diff
@@ -302,223 +302,223 @@ amp-story-grid-layer * {
-  grid-gap: 16px;
+  grid-gap: 16px !important;
   grid-template-rows: 100% !important;
-  justify-content: start;
-  padding: 32px;
+  justify-content: start !important;
+  padding: 32px !important;
 }
 
 .i-amphtml-story-grid-template-thirds {
   grid-template-rows: repeat(3, 1fr) !important;
   grid-template-areas: "upper-third"
                        "middle-third"
                        "lower-third" !important;
-  padding: 32px;
+  padding: 32px !important;
 }
 
 /** Bookend */
 .i-amphtml-story-bookend {
   background: rgba(0, 0, 0, 0.85) !important;
-  color: #fff;
+  color: #fff !important;
   bottom: 0 !important;
   display: none !important;
   left: 0 !important;
-  position: absolute;
+  position: absolute !important;
   right: 0 !important;
   top: 0 !important;
-  padding: 44px 32px 32px;
-  overflow: auto;
-  z-index: 10;
+  padding: 44px 32px 32px !important;
+  overflow: auto !important;
+  z-index: 10 !important;
 }
 
 .i-amphtml-story-bookend-active .i-amphtml-story-bookend {
   display: block !important;
 }
 
 .i-amphtml-story-bookend-heading {
-  text-transform: uppercase;
-  font-size: 10px;
-  padding: 0 0 8px;
-  margin: 48px 0 8px;
-  border-bottom: 1px solid rgba(255, 255, 255, 0.25);
-  color: rgba(255, 255, 255, 0.54);
-  font-family: 'Roboto', sans-serif;
-  font-weight: 700;
-  letter-spacing: 0.83px;
-  line-height: 1;
+  text-transform: uppercase !important;
+  font-size: 10px !important;
+  padding: 0 0 8px !important;
+  margin: 48px 0 8px !important;
+  border-bottom: 1px solid rgba(255, 255, 255, 0.25) !important;
+  color: rgba(255, 255, 255, 0.54) !important;
+  font-family: 'Roboto', sans-serif !important;
+  font-weight: 700 !important;
+  letter-spacing: 0.83px !important;
+  line-height: 1 !important;
 }
 
 .i-amphtml-story-bookend-article-meta {
-  font-family: 'Roboto', sans-serif;
-  font-weight: 300;
-  font-size: 10px;
-  line-height: 1;
-  color: rgba(255, 255, 255, 0.54);
+  font-family: 'Roboto', sans-serif !important;
+  font-weight: 300 !important;
+  font-size: 10px !important;
+  line-height: 1 !important;
+  color: rgba(255, 255, 255, 0.54) !important;
 }
 
 .i-amphtml-story-bookend-article {
-  display: block;
-  text-decoration: none;
+  display: block !important;
+  text-decoration: none !important;
   color: inherit;
-  margin: 24px 0;
-  overflow: hidden;
+  margin: 24px 0 !important;
+  overflow: hidden !important;
 }
 
 .i-amphtml-story-bookend-article-set:last-child >
     .i-amphtml-story-bookend-article:last-child {
-  margin-bottom: 0; /* bottom of bookend has padding > article margin */
+  margin-bottom: 0 !important; /* bottom of bookend has padding > article margin */
 }
 
 .i-amphtml-story-bookend-article-heading {
-  font-family: 'Roboto', sans-serif;
-  font-weight: 300;
-  font-size: 16px;
-  color: #ffff;
+  font-family: 'Roboto', sans-serif !important;
+  font-weight: 300 !important;
+  font-size: 16px !important;
+  color: #ffff !important;
```

Comment:
nit: can you also drop the extra `f` from this line?

Reply:
Done